python/flare/ironstrings/strings.py

```diff
@@ -8,8 +8,8 @@
 
 
 ASCII_BYTE = " !\"#\$%&\'\(\)\*\+,-\./0123456789:;<=>\?@ABCDEFGHIJKLMNOPQRSTUVWXYZ\[\]\^_`abcdefghijklmnopqrstuvwxyz\{\|\}\\\~\t"
-ASCII_RE_4 = re.compile("([%s]{%d,})" % (ASCII_BYTE, 4))
-UNICODE_RE_4 = re.compile(b"((?:[%s]\x00){%d,})" % (ASCII_BYTE, 4))
+ASCII_RE_4 = re.compile(b"([%s]{%d,})" % (ASCII_BYTE.encode(), 4))
+UNICODE_RE_4 = re.compile(b"((?:[%s]\x00){%d,})" % (ASCII_BYTE.encode(), 4))

```

Comment:
recommend an explicit encoding here, probably `ascii`

Reply:
yes, please use `r" !\"#\$...~\t".encode("ascii")`, then good to go, thanks!